uk_bin_collection/uk_bin_collection/councils/ClackmannanshireCouncil.py

```
@@ -0,0 +1,93 @@
+                        collection_date = event.start.date().strftime(date_format)
+                        key = (bin_type, collection_date)
+                        if key not in seen:
+                            seen.add(key)
+                            data["bins"].append(
+                                {
+                                    "type": bin_type,
+                                    "collectionDate": collection_date,
+                                }
+                            )
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)